code/data_structures/src/stack/stackUsingLinkedList/stackUsingLinkedList.cpp

```diff
@@ -0,0 +1,90 @@
+#include<iostream>
+#include<malloc.h>
+using namespace std;
+struct node
+{
+    int data;
+    struct node *next;
+};
+void Traverse_stack(struct node* top)
+{
+    struct node* temp=top;
+    cout<<"Current Stack->\n";
+    while(temp!=NULL)
+    {
+    cout<<temp->data<<"\n";
+    temp=temp->next;
+    }
+}
+struct node * push(struct node * top,int data)
+{
+    struct node * temp;
+    temp=(struct node*)malloc(sizeof(struct node));
+    temp->data=data;
+    temp->next=top;
+    top=temp;
+    cout<<top->data<<" is pushed into the stack\n";
```

Comment:
Don't output in operations, only when explicitly printing.

Reply:
OK..